pkg/users/group-details.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState } from 'react';
+import { superuser } from "superuser";
+import { apply_modal_dialog } from "cockpit-components-dialog.jsx";
+import { show_unexpected_error } from "./dialog-utils.js";
+
+import {
+    Button,
+    Card, CardBody, CardHeader, CardTitle, CardActions,
+    Flex, HelperText, HelperTextItem,
+    Label, LabelGroup,
+    Page, PageSection,
+    Select, SelectOption, SelectVariant, Text, TextVariants, Breadcrumb, BreadcrumbItem,
+    Form, FormGroup,
+} from '@patternfly/react-core';
+import { UndoIcon } from '@patternfly/react-icons';
+
+import { GroupActions } from "./group-actions.jsx";
+
+const _ = cockpit.gettext;
+
+export function GroupDetails({ groups, accounts, group }) {
+    const current_group = groups.find(g => g.name == group);
+    return (
+        <Page groupProps={{ sticky: 'top' }}
+              isBreadcrumbGrouped
+              id="group"
+              breadcrumb={
+                  <Breadcrumb>
+                      <BreadcrumbItem to="#/">{_("Groups")}</BreadcrumbItem>
+                      <BreadcrumbItem isActive>{group}</BreadcrumbItem>
+                  </Breadcrumb>}>
+            <PageSection>
+                <Card className="group-details" id="group-details">
+                    <CardHeader>
+                        <CardTitle id="group-title"><Text component={TextVariants.h2}>{group}</Text></CardTitle>
+                        {superuser.allowed && <CardActions>
+                            <GroupActions group={current_group} accounts={accounts} />
+                        </CardActions>}
```

Comment:
I prepared a clean fedora-37 directly from your branch, started it, and went to https://127.0.0.2:9092/users#/group/wheel . The actions kebab is empty:

![image](https://user-images.githubusercontent.com/200109/222061619-9e0d0fd8-1cc3-450f-a81b-4ae414c7c606.png)

Same happens for "nogroup" and all other system groups. It only shows "Rename" and "Delete" for "user" groups like "admin":

![image](https://user-images.githubusercontent.com/200109/222061797-845e3eff-6009-4baa-b90c-2e50b4af692b.png)

I.e. please hide the kebab if it doesn't have any actions.